src/Microsoft.Kusto.ServiceLayer/DataSource/DataSourceIntellisense/KustoIntellisenseHelper.cs

```diff
@@ -23,179 +20,28 @@ namespace Microsoft.Kusto.ServiceLayer.DataSource.DataSourceIntellisense
     /// </summary>
     public class KustoIntellisenseHelper
     {
-        /// <summary>
-        /// Convert CLR type name into a Kusto scalar type.
-        /// </summary>
-        private static ScalarSymbol GetKustoType(string clrTypeName)
```

Comment:
Moved all of this to the KustoIntelisenseClient since it's related to that and not accessed anywhere else.